handlers/confirmSigHandler.go

```diff
@@ -0,0 +1,55 @@
+package handlers
+
+import (
+	"github.com/FourthState/plasma-mvp-sidechain/msgs"
+	"github.com/FourthState/plasma-mvp-sidechain/store"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// NewSpendHandler sets the inputs of a spend msg to spent and creates new
+// outputs that are added to the data store.
+func NewConfirmSigHandler(ds store.DataStore) sdk.Handler {
+	return func(ctx sdk.Context, msg sdk.Msg) sdk.Result {
+		confirmSigMsg, ok := msg.(msgs.ConfirmSigMsg)
+		if !ok {
+			panic("Msg does not implement SpendMsg")
+		}
+
+		/* Get tx, update tx data with confirm sigs, and store updated tx object */
+		// TODO: Validation on correctness of confirm sigs?
+		tx1, ok := ds.GetTxWithPosition(ctx, confirmSigMsg.Input1.Position)
+		if !ok {
+			panic("no transaction exists for the position provided")
+		}
+
+		for i := 0; i < len(tx1.Transaction.Inputs); i++ {
+			// TODO: Is TxIndex sufficient? Also should likely check output index?
+			if tx1.Transaction.Inputs[i].Position.TxIndex != confirmSigMsg.Input1.Position.TxIndex {
```

Comment:
can we add an `Equals` method to `Input` and simply to `if tx1.Transaction.Input[i].Position.Equals(...)` { do the thing}.

Typically do not want to use `continue`. Should wrap the state updates within the `if` block for when it should happen. Makes it easier to read.